query-service-impl/src/main/java/org/hypertrace/core/query/service/prometheus/PrometheusRestClient.java

```diff
@@ -125,12 +123,12 @@ public OkHttpResponseCallback(Request request) {
     }
 
     @Override
-    public void onResponse(Call call, Response response) throws IOException {
+    public void onResponse(@NonNull Call call, @NonNull Response response) {
```

Comment:
why do we need explicitly `@NonNull`?

Reply:
IDE suggestion, it says `Not annotated parameter overrides @NotNull parameter `, basically this param is marked not null in the parent